editor/store/browser.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { throttle } from 'lodash';
+
+/**
+ * Enhance a redux store with the browser size
+ *
+ * @param {Object} store Redux Store
+ */
+function enhanceWithBrowserSize( store ) {
+	const updateSize = throttle( () => {
```

Comment:
Minor: maybe in this use case lodash debounce is a better option https://lodash.com/docs/4.17.4#debounce. Because we are interested in minimizing the number of actions dispatched and we are just interested in the final window size after resize events stop happening.

Reply:
In that case, the Gutenberg UI would only show a resize until after you stop completely with resizing. With throttle, you see UI feedback before you've ended your resize.

Reply:
Even with debounce, we get some feedback from the browser, e.g: CSS breakpoints. We don't get the feedback of breakpoints implemented in javascript like hiding things or change components. As long as our redraws are efficient I think the throttle is ok. So I'm fine with both versions.